users/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import re
+import requests
 
-# Create your views here.
+from twilio.rest  import Client
+from django.views import View
+from django.http  import JsonResponse
+from random       import randint
+from users.models import User, PhoneCheck
+from my_settings  import ACCOUNT_SID, AUTH_TOKEN
+
+class SendSmSView(View):
+    def post(self, request):
+        data         = json.loads(request.body)
+        input_mobile = data.get('mobile')
+        try:
+            if not input_mobile:
+                return JsonResponse({'message': 'KEY_ERROR'}, status=401)
+            if re.match(r"(010)\d{4}\d{4}",input_mobile):
+                to_mobile = '+82'+ input_mobile[1:]
+            else:
+                return JsonResponse({'message':'INVALID_PHONE_NUMBER'}, status=401)
```

Comment:
```suggestion            
            return JsonResponse({'message':'INVALID_PHONE_NUMBER'}, status=401)
```
안해도 되는 else는 줄여서 한줄이라도 줄입시다.

Reply:
수정 진행 하였습니다.